MekHQ/src/mekhq/gui/dialog/PartsReportDialog.java

```diff
@@ -239,24 +258,73 @@ public void actionPerformed(ActionEvent e) {
         new PartsInUseTableModel.ButtonColumn(overviewPartsInUseTable, addInBulk,
                 PartsInUseTableModel.COL_BUTTON_GMADD_BULK);
 
-        overviewPartsPanel.add(new JScrollPane(overviewPartsInUseTable), BorderLayout.CENTER);
 
-        JPanel panButtons = new JPanel(new GridBagLayout());
+        JScrollPane tableScroll = new JScrollPane(overviewPartsInUseTable);
+
+        ignoreMothballedCheck = new JCheckBox(resourceMap.getString("chkIgnoreMothballed.text"));
+        ignoreMothballedCheck.addActionListener(evt -> refreshOverviewPartsInUse());
+
+        boolean reverse = campaign.getCampaignOptions().isReverseQualityNames();
+        String[] qualities = {
+            " ", // Combo box is blank for first one because it accepts everything and is default
+            PartQuality.QUALITY_B.toName(reverse),
+            PartQuality.QUALITY_C.toName(reverse),
+            PartQuality.QUALITY_D.toName(reverse),
+            PartQuality.QUALITY_E.toName(reverse),
+            PartQuality.QUALITY_F.toName(reverse)
+        };
+
+        ignoreSparesUnderQualityCB = new JComboBox<String>(qualities);
+        ignoreSparesUnderQualityCB.setMaximumSize(ignoreSparesUnderQualityCB.getPreferredSize());
+        ignoreSparesUnderQualityCB.addActionListener(evt -> refreshOverviewPartsInUse());
+        JLabel ignorePartsUnderLabel = new JLabel(resourceMap.getString("lblIgnoreSparesUnderQuality.text"));
+
         JButton btnClose = new JButton("Close");
         btnClose.addActionListener(evt -> setVisible(false));
-        panButtons.add(btnClose, new GridBagConstraints());
-        overviewPartsPanel.add(panButtons, BorderLayout.PAGE_END);
+        
+        layout.setHorizontalGroup(layout.createParallelGroup()
+            .addComponent(tableScroll)
+            .addGroup(layout.createSequentialGroup()
+                .addGroup(layout.createSequentialGroup()
+                    .addPreferredGap(LayoutStyle.ComponentPlacement.RELATED, GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+                    .addComponent(ignorePartsUnderLabel)
+                    .addComponent(ignoreSparesUnderQualityCB)
+                    .addComponent(ignoreMothballedCheck)
+                    .addPreferredGap(LayoutStyle.ComponentPlacement.RELATED, GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+                    .addComponent(btnClose))));
 
-        this.setLayout(new BorderLayout());
-        this.add(overviewPartsPanel, BorderLayout.CENTER);
-        setPreferredSize(new Dimension(1000, 800));
+        layout.setVerticalGroup(layout.createSequentialGroup()
+            .addComponent(tableScroll)
+            .addGroup(layout.createParallelGroup(Alignment.BASELINE)
+                .addComponent(ignoreMothballedCheck)
+                .addComponent(ignorePartsUnderLabel)
+                .addComponent(ignoreSparesUnderQualityCB)
+                .addComponent(btnClose)));
+
+        setPreferredSize(new Dimension(1400, 1000));
```

Comment:
I forgot to mention, this doesn't actually do what you think it does, from what Juli told me the other day, you need to include an override. Lemme grab the details. This is just a quick message to alert you to the issue, I'll follow up in a second.